Android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.4.0'
```

Comment:
why this change?
Is there any reason for downgrading the version?

Reply:
@Florence-Njeri Revert this change. Then only, I'll be able to approve and merge it.

Reply:
Revert this change